### ISSUE TITLE
Error : require() of ES Module .../ag-charts-community/dist/esm/es6/main.mjs not supported.

### ISSUE THREAD

**Issue by Tejada-Joyce** [ag-grid/ag-grid#6873]
**I'm submitting a ...**  (check one with "x")
```
[] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[x] support request => see 'Requesting Community Support'
```

* **AG Grid version:** 
  -  "ag-charts-community": "8.0.3",
  -   "ag-charts-react": "8.0.3",
  -  "ag-grid-community": "30.0.3",
  -  "ag-grid-react": "30.0.4",

* **Language:** TypeScript 4.9.5

The error I'm getting when testing (Vitest) is this one:

```
Error: require() of ES Module [...]/node_modules/ag-charts-community/dist/esm/es6/main.mjs not supported.
Instead change the require of [...]/node_modules/ag-charts-community/dist/esm/es6/main.mjs to a dynamic import() which is available in all CommonJS modules.
```

I was previously getting the error mentioned here: https://github.com/ag-grid/ag-grid/issues/6821, but after upgrading the packages versions, I started seeing this error.
I have read your documentation about testing. There is not something specific for Vitest, but since it's pretty similar to Jest, I assumed that applied to Vitest. By the way, is the [documentation](https://www.ag-grid.com/javascript-data-grid/testing/) still valid? I'm asking since the other [issue](https://github.com/ag-grid/ag-grid/issues/6821) I mentioned before seems to suggest something different. I'm just trying to figure out what I am doing wrong, so I would really appreciate your help here. 

Thanks!

**Comment by seanlandsman** [comment]
Hi - are you able to provide a repo I can take a look at for this?

**Comment by Tejada-Joyce** [comment]
I put this together in codesandbox: https://codesandbox.io/p/sandbox/vite-react-ts-forked-frppwv?file=%2Fvite.config.ts%3A1%2C1 . It's a simple example, but I get the same error when running the test.

**Comment by terlan4** [comment]
Hi, @Tejada-Joyce 
Did you manage to solve the issue? I have the same problem with Next.js. Please share your solution if you managed to solve. Thanks

**Comment by terlan4** [comment]
@seanlandsman I have the same issue while using with Next.js. May I know if there is anything new added to docs to solve this?
https://github.com/ag-grid/ag-grid/issues/6912

**Comment by seanlandsman** [comment]
We'll be releasing 30.0.6 shortly that should resolve these issues - we apologise for the hassle and noise that this has caused

**Comment by seanlandsman** [comment]
We have just released 30.0.6 : please see [ES Modules & CommonJS](https://blog.ag-grid.com/ag-grid-es-modules-and-commonjs/) for more information
